PopPUNK/__main__.py

```diff
@@ -116,6 +116,8 @@ def get_options():
     modelGroup.add_argument('--min-cluster-prop', help='Minimum proportion of points in a cluster '
                                                         'in DBSCAN fitting [default = 0.0001]', type=float, default=0.0001)
     modelGroup.add_argument('--threshold', help='Cutoff if using --fit-model threshold', type=float)
+    modelGroup.add_argument('--no-plot', help='Switch off plotting, which can be slow for large datasets', type=bool,
```

Comment:
I think we should move this option and `--no-local` into the other group, as this keeps all the optimisation arguments together

Reply:
Done in c450710 - might apply `no-plot` to other plots as well?

Reply:
Applied it to scatter plot in case useful in 03f11a8.